pontoon/base/static/js/translate.js

```diff
@@ -2151,7 +2151,11 @@ var Pontoon = (function (my) {
         submittedPluralForm = 0;
       }
 
-      return $.ajax({
+      if (self.XHRupdateOnServer) {
+        self.XHRupdateOnServer.abort();
```

Comment:
True. But in this case (if the server already processed the call) we no longer have the problem we are solving - which is being able to submit two approved translations for the same string for the same locale.